README.md

```diff
@@ -56,6 +56,7 @@ In order to deploy both the stacks the user needs to provide a set of required a
 | mlInstanceType         | Optional    | string  | EC2 instance type for ml node. Defaults to r5.xlarge. See options in `lib/opensearch-config/node-config.ts` for available options. E.g., `-c mlInstanceType=m5.xlarge`                                                                                                                           |
 | jvmSysProps            | Optional    | string  | A comma-separated list of key=value pairs that will be added to `jvm.options` as JVM system properties.                                                                                                                                                                                          |
 | additionalConfig       | Optional    | string  | Additional opensearch.yml config parameters passed as JSON. e.g., `--context additionalConfig='{"plugins.security.nodes_dn": ["CN=*.example.com, OU=SSL, O=Test, L=Test, C=DE", "CN=node.other.com, OU=SSL, O=Test, L=Test, C=DE"], "plugins.security.nodes_dn_dynamic_config_enabled": false}'` |
+| additionalOSDconfig       | Optional    | string  | Additional opensearch_dashboards.yml config parameters passed as JSON. e.g., `additionalOSDconfig='{"data.search.usageTelemetry.enabled": "true"}'` |
```

Comment:
Nit: maybe `additionalOsdconfig` to conform with camel-casing?

Reply:
Fixed everywhere!